lib/_emerge/actions.py

```diff
@@ -1899,6 +1902,10 @@ def action_info(settings, trees, myopts, myfiles):
 				if default is not None and \
 					default == v:
 					continue
+
+				if k in filter_vars:
+					v = re.sub(r'//(.+):.+@(.+)', r'//\1:*password*@\2', v)
+
```

Comment:
Maybe we can just import the _hide_url_passwd function and use it here, and I'm thinking we should use it on all the variables just in case.